var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -162,3 +162,37 @@ def cmake_args(self):
                 args.append("-DCMAKE_CUDA_FLAGS=-arch sm_{0}".format(cuda_arch[0]))
 
         return args
+
+    def test_smoke_test(self):
+        """Build and run ctests"""
+        spec = self.spec
+        smoke_test_dir = self.test_suite.current_test_data_dir.tests
+        with working_dir(smoke_test_dir):
+            with open(join_path(smoke_test_dir, "/CMakeLists.txt"), "w") as f:
```

Comment:
You can skip the `join_path` since you want the file in the current directory:

```suggestion
            with open("CMakeLists.txt", "w") as f:
```